.eslintrc.json

```diff
@@ -22,7 +22,7 @@
                 "MemberExpression": 2,
                 "ignoredNodes": [ "JSXAttribute" ]
             }],
-        "newline-per-chained-call": ["error", { "ignoreChainWithDepth": 2 }],
+        "newline-per-chained-call": ["error", { "ignoreChainWithDepth": 3 }],
```

Comment:
What would be a legit use case for that? In a lot of cases even two is too much. In particular a `.then()` should never be chained if the same statement also has a `.catch()` or `.finally()`.

Reply:
See the commit message about that one: I agree that for the case of chaining promise callbacks this is a bad idea, but there are other cases (like `.strip.split.join.etc()` type things) where this isn't so bad.